packages/appeals-service-api/__tests__/unit/lib/notify.test.js

```diff
@@ -105,19 +110,14 @@ describe('lib/notify', () => {
 			await sendSubmissionReceivedEmailToLpa(householderAppeal);
 
 			expect(NotifyBuilder.reset).toBeCalled();
-			expect(NotifyBuilder.reset().setTemplateId).toBeCalledWith('lpa-template');
+			expect(NotifyBuilder.reset().setTemplateId).toBeCalledWith('appellant-template');
 			expect(NotifyBuilder.reset().setTemplateId().setDestinationEmailAddress).toBeCalledWith(
-				'AppealPlanningDecisionTest@planninginspectorate.gov.uk'
+				'test@pins.com'
```

Comment:
The domain `pins.com` exists but is entirely unrelated to the planning inspectorate so we should stick to either a non-existent email address ending `@planninginspectorate.gov.uk` or an email address ending with `@example.com` (as example.com is a reserved domain).

Reply:
It looks like we're using this @pins.com convention throughout the test fixtures on the appeals service api